recipe/meta.yaml

```diff
@@ -58,17 +57,16 @@ requirements:
    - cspice
    - csm>=3.0.3,<3.0.4
    - curl
+   - cxx-compiler=1.1.2
    - doxygen
    - eigen<3.4
-   # versioned due to major release of embree 3 on conda-forge
-   # it is not compatible without changes to ISIS
-   - embree>=2.17, <3
+   - embree>=2.17,<3
    - ffmpeg
    - geos>=3.7,<3.8
    - geotiff
    - gmp
    - graphviz
-   - gsl>=2.6,<2.7
```

Comment:
From my understanding this pin needs to stay. See #4987

Reply:
Should I make this change in the environment.yml too?